nbs/models.hint.ipynb

```diff
@@ -17,18 +17,27 @@
    ]
```

Comment:
<p>I can't see the plot in the ReviewNB. Just be sure that is included. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
We skip the execution of the notebooks code to avoid computation, the plots remain normally as placeholders.